app/components/questions/RideshareTripsOHare.js

```diff
@@ -0,0 +1,135 @@
+import { useState, useEffect } from 'react'
+import useFetch from 'use-http'
+import { Table } from '../../components/Table'
+import { Notification } from '../../components/Notification'
+import { FailureNotification } from '../../components/Notification'
+import {
+  Formatter,
+  calculatePercentChange,
+} from '../../site/metrics'
+
+const RIDESHARE_ENDPOINT = `${process.env.NEXT_PUBLIC_API}/question/ohare/rideshare`;
+  
+const MAX_PCT_CHANGE = -0.76;
+const RIDESHARE_DROPOFF_ALL = [
+  {
+    key: "dropoff_community_area",
+    name: "Area Number",
+  },
+  {
+    key: "area_name",
+    name: "Community Area",
+  },
+  {
+    key: "total_trips_2019",
+    name: "Dropoffs in 2019",
+    format: Formatter.numberWithCommas,
+    rowClasses: ["right"],
+  },
+  {
+    key: "total_trips_2020",
+    name: "Dropoffs in 2020",
+    format: Formatter.numberWithCommas,
+    rowClasses: ["right"],
+  },
+  {
+    key: "pct_change",
+    name: "Change",
+    format: Formatter.percentChangeWithNoDecimal,
+    rowClasses: ["right"],
+    rgb: "209, 31, 0",
+    alpha: (v) => (v / MAX_PCT_CHANGE),
+  },
+];
+
+function addTotalTripsYear(before,since) {
+
+  //retrieving key:value pairs
+  var dropoffs = before.map(({
+    total_trips: total_trips_2019,
+    dropoff_community_area: dropoff_community_area,
+    area_name: area_name
+  }) => ({
+    total_trips_2019,
+    dropoff_community_area,
+    area_name: area_name
+  }));
+  
+  const dropoffs_2020 = since.map(({
+    total_trips: total_trips_2020,
+
+  }) => ({
+    total_trips_2020,
+
+  }));
+  
+  //adding 2020 trips within the same object as the 2019 trips
+  for (var i = 0; i < dropoffs.length ; i++){
```

Comment:
We still have the issue here of assuming the data is (a) sorted and (b) has all the community areas with no nulls. How do you want to handle it?

Do you want to solve it on the frontend or on the backend?

If you go with a frontend solution:
- Make a dictionary and use area number as a key
- Then convert the dictionary to a list
- Similar to what you did when fixed the backend code

If you go with a backend solution:
- You could rewrite the SQL query to return data in the format you want
- Or you could use Python to merge the results, again similar to what you did to fix the other problem